.circleci/config.yml

```diff
@@ -77,53 +51,113 @@ jobs:
           git config user.email "m.a.swertz@rug.nl"
           git config user.name "mswertz"
           git config url.https://.insteadOf git://
+    - persist_to_workspace:
+          root: .
+          paths:
+              - .
+    
+  build:
+    docker:
+    - image: molgenis/ci-build:latest
+    - image: postgres:15-alpine
+      environment:
+        POSTGRES_USER: postgres
+        POSTGRES_PASSWORD: postgres
+        POSTGRES_DB: postgres
+    working_directory: ~/repo
+    
+    # The resource_class feature allows configuring CPU and RAM resources for each job. Different resource classes are available for different executors. https://circleci.com/docs/2.0/configuration-reference/#resourceclass
+    resource_class: large
+
+    environment:
+      JVM_OPTS: -Xmx3200m
+      GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
+      TERM: dumb
+
+    steps:
+    - attach_workspace:
+        at: ~/repo
 
+    - run: ./gradlew clean assemble testClasses -x :apps:testJavaScript
```

Comment:
what does 'testClasses' do?

Reply:
https://stackoverflow.com/questions/43638697/gradle-compile-tests-but-do-not-run-them